.github/workflows/release.yml

```diff
@@ -86,6 +86,10 @@ jobs:
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
       - name: build-amd64
         run: "docker buildx build --platform linux/amd64 -t ${{ github.repository }}:${{ env.VERSION }} --push --cache-from type=registry,ref=${{ github.repository }} --cache-to type=registry,ref=${{ github.repository }},mode=max ."
+      - name: buildkit-amd64
+        run: docker buildx build -f Dockerfile-buildkit --target rootless --platform linux/amd64 -t ${{ github.repository }}:${{ env.VERSION }}-build --push --cache-from type=registry,ref=${{ github.repository }} --cache-to type=registry,ref=${{ github.repository }},mode=max .
+      - name: buildkit-amd64-privileged
+        run: docker buildx build -f Dockerfile-buildkit --target privileged --platform linux/amd64 -t ${{ github.repository }}:${{ env.VERSION }}-build-privileged --push --cache-from type=registry,ref=${{ github.repository }} --cache-to type=registry,ref=${{ github.repository }},mode=max .
```

Comment:
What's the difference between `buildkit-amd64` and `buildkit-amd64-privileged`?

Reply:
the `privileged` variation uses a different target `--target privileged`. The privileged image is a bit less secure than the ` buildkit-amd64` because it runs as root and need some additional escalation privileges.
I built both because the preferred one should be `buildkit-amd64` but not all cloud providers support that version, probably because of differences in the underlying work nodes configuration.
`privileged` is just the standard nomenclature in Buildkit's repostiory